MobileCenterCrashes/MobileCenterCrashes/Internals/MSWrapperExceptionManagerInternal.h

```diff
@@ -10,34 +10,50 @@
 @property NSMutableDictionary *wrapperExceptionData;
 @property NSData *unsavedWrapperExceptionData;
 @property CFUUIDRef currentUUIDRef;
-@property(weak, nonatomic) id<MSWrapperCrashesInitializationDelegate> crashesDelegate;
+@property(weak, nonatomic) id <MSWrapperCrashesInitializationDelegate> crashesDelegate;
```

Comment:
We should have docs for properties and methods in the header. It was done by @achocron.